ceph-multibranch-pipeline/config/definitions/ceph-multibranch-pipeline.yml

```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: ceph-multibranch-pipeline
+    project-type: multibranch
+    number-to-keep: 600
```

Comment:
Will this job contain the console logs for the entirety of `make check` and `API tests` jobs?  If so, 600 is far too many.

Reply:
No problem. I'll reduce it to 300 (but keep the 60 day, so it's limited by amount rather than time).